07-performance.Rmd

```diff
@@ -80,7 +80,7 @@ An additional quirk of `ifelse()` is that although it is more *programmer effici
 ```{r}
 marks = runif(n = 10e6, min = 30, max = 99)
```

Comment:
This is probably too picky, but should marks be rounded or generated via sample?